### ISSUE TITLE
Initial locale argument was not passed into serverSideTranslations

### ISSUE THREAD

**Issue by uniquesolution29** [isaachinman/next-i18next#1133]
## Describe the bug
It works locally.
However, it will make nextServer internal error once I deployed it on firebase.

## Occurs in next-i18next version
8.1.3 must provide a clear and concise description of what you expected to happen.

## Config
```
module.exports = {
  i18n: {
    defaultLocale: 'en',
    locales: ['en', 'ko'],
  },
};
```

## Screenshots
![image](https://user-images.githubusercontent.com/48445639/114059078-ae359780-989c-11eb-97b4-a17195cce7d6.png)

**Comment by isaachinman** [comment]
Hi @klimazarin – unfortunately I'm unable to provide support for platform-specific issues. Can you provide any more details?

**Comment by klimazarin** [comment]
Thanks for your quick response.

You can see my app here: https://app.wedeo.io
However, it doesn't work at all :)
The problem is why the error appears on a production mode.

I just followed the sample code, if you want to see.
```
// _app.tsx
import { appWithTranslation } from 'next-i18next';

const App = ({ Component, pageProps }: AppProps): JSX.Element => {
  return (
    <Provider store={store}>
      <MainWrapper>
        <Component {...pageProps} />
      </MainWrapper>
    </Provider>
  );
};

export default appWithTranslation(App);

// code snippets regarding serverSideRendering

export const getStaticProps: any = async ({ locale }) => ({
  props: {
    ...(await serverSideTranslations(locale, [])),
  },
});
// or
export const getServerSideProps: GetServerSideProps = async (context) => {
  const { teamId, email } = context.query;
  let teamName;

  if (!teamId) {
    return { props: {} };
  }

  if (teamId) {
    teamName = await getTeamName(teamId as string);
  }

  return {
    props: {
      teamId,
      teamName,
      email: email || '',
      ...(await serverSideTranslations(context.locale, [])),
    },
  };
};

export default withTranslation('auth')(SignUpPage);

**Comment by isaachinman** [comment]
Is the error reproducible if you run the app in prod mode locally? Eg `yarn build && yarn start`?

**Comment by klimazarin** [comment]
App works in prod mode locally.
The error only occurs on cloud server.

**Comment by isaachinman** [comment]
Then unfortunately I'm unable to help, as whatever error you are experiencing is specific to the platform.

**Comment by thomb** [comment]
@uniquesolution29 Just a heads up I was running into this issue when building my next,js app in a docker container, which not exactly similar I thought to pass on how I resolved this.

I was copying the output of `next build` - i.e. `.next` - into my final build location, but was not copying over the `next.config.js` not `next-i18n.config.js`. By manually copying those files over into the final build location I was able to run the next app with translations without issue.

Again, not certain if this is directly relevant to your build issue but perhaps it provides you some avenue in to debugging it.
